var/spack/repos/builtin/packages/r-ggally/package.py

```diff
@@ -14,15 +14,18 @@ class RGgally(RPackage):
        group pairwise plot matrix, a parallel coordinates plot, a survival
        plot, and several functions to plot networks."""
 
-    homepage = "https://cran.r-project.org/package=GGally"
-    url      = "https://cran.r-project.org/src/contrib/GGally_1.3.2.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/GGally"
+    homepage = "https://cloud.r-project.org/package=GGally"
+    url      = "https://cloud.r-project.org/src/contrib/GGally_1.3.2.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/GGally"
 
+    version('1.4.0', sha256='9a47cdf004c41f5e4024327b94227707f4dad3a0ac5556d8f1fba9bf0a6355fe')
     version('1.3.2', 'efe58133ba8431198af7afb6bcb76264')
 
-    depends_on('r-ggplot2', type=('build', 'run'))
-    depends_on('r-gtable', type=('build', 'run'))
-    depends_on('r-plyr', type=('build', 'run'))
+    depends_on('r@3.1:', type=('build', 'run'))
+    depends_on('r-ggplot2@2.2.0:', type=('build', 'run'))
+    depends_on('r-gtable@0.2.0:', type=('build', 'run'))
+    depends_on('r-plyr@1.8.3:', type=('build', 'run'))
     depends_on('r-progress', type=('build', 'run'))
     depends_on('r-rcolorbrewer', type=('build', 'run'))
-    depends_on('r-reshape', type=('build', 'run'))
+    depends_on('r-reshape@0.8.5:', type=('build', 'run'))
+    depends_on('r-rlang', when='@1.4.0:', type=('build', 'run'))
```

Comment:
> SystemRequirements:	openssl

Reply:
Needed for version 1.4.0 and above; added.